src/app/account-statement/account-statement.component.html

```diff
@@ -1,144 +1,194 @@
 <mat-spinner [diameter]="60" *ngIf="totalRecords === null"></mat-spinner>
 <div class="d-flex w-100">
```

Comment:
```suggestion
<div class="account-statement d-flex w-100">
```

account-statement make this class and write all css under this in css file